python/lsst/ts/logging_and_reporting/all_sources.py

```diff
@@ -183,7 +186,7 @@ def dayobs_range(self):
 
     # This will have to be async def night_tally_observation_gaps
     # if efd_src is used.
-    def night_tally_observation_gaps(self, verbose=False):
+    def SAVE_night_tally_observation_gaps(self, verbose=False):
```

Comment:
```suggestion
    def save_night_tally_observation_gaps(self, verbose=False):
```

Reply:
If applying the suggestion please remember to update calls to the function too :pray:.